quality-tool/README.md

```diff
@@ -0,0 +1,23 @@
+## Install
+
+Clone the repo and install dependencies:
+
+```bash
+yarn
+```
```

Comment:
do indicate any where what version of yarn?

Reply:
Fixed :)